README.md

```diff
@@ -4,7 +4,7 @@
   <a href="https://build.ros2.org/job/Hdev__openarm__ubuntu_jammy_amd64/" alt="alt="ROS2 build status">
     <img src="https://build.ros2.org/buildStatus/icon?job=Hdev__openarm__ubuntu_jammy_amd64"/>
   </a>
-  <a href="https://github.com/reazon-research/openarm_ros2/tree/main/LICENSE" alt="BSD-3-Clause License">
+  <a href="https://github.com/reazon-research/openarm_ros2/tree/main/LICENSE" alt="Apache 2.0 License">
```

Comment:
```suggestion
  <a href="https://github.com/reazon-research/openarm_ros2/tree/main/LICENSE.txt" alt="Apache 2.0 License">
```

Reply:
I renamed this license file name from `LICENSE.txt` to `LICENSE` because ROS projects commonly use this name.
- https://github.com/ros2/ros2_tracing 
- https://github.com/ros-infrastructure/rosindex
- https://github.com/b-robotized/ros_team_workspace

Reply:
Could you do it in all `LICENSE.txt`s?

Reply:
fix: [6921635](https://github.com/reazon-research/openarm_ros2/pull/13/commits/692163545b64a13b020e93cbaed68f61e9de6b02) I'm sorry I missed them.